src/components/reusable/table/table-row.ts

```diff
@@ -194,7 +225,7 @@ export class TableRow extends LitElement {
         ? html`
             <kyn-td .align=${'center'} ?dense=${this.dense}>
               <kyn-checkbox
-                ?disabled=${this.disabled}
+                ?disabled=${this.disabled || this.locked}
                 .checked=${this.selected}
```

Comment:
`disabled` and `locked` same ? What's the difference ?

Reply:
I have the description of each prop in details in code and on Storybook, too.
So, basically a disabled and locked are similar, except for that a locked row will have a different bottom-border. And locked is specially used for column setting feature.